ui/app/routes/vault/cluster/dashboard.js

```diff
@@ -9,10 +9,32 @@ import { hash } from 'rsvp';
 
 export default class VaultClusterDashboardRoute extends Route {
   @service store;
+  @service version;
+
+  async getVaultConfiguration() {
+    try {
+      const adapter = this.store.adapterFor('application');
+      const configState = await adapter.ajax('/v1/sys/config/state/sanitized', 'GET');
+      return configState.data;
+    } catch (e) {
+      return e.httpStatus;
+    }
+  }
 
   model() {
     return hash({
       secretsEngines: this.store.query('secret-engine', {}),
+      version: this.version,
+      vaultConfiguration: this.getVaultConfiguration(),
     });
   }
+
+  setupController(controller, resolvedModel) {
+    super.setupController(controller, resolvedModel);
+    const { version } = resolvedModel;
+
+    controller.versionHeader = version.isEnterprise
+      ? `Vault v${version.version.slice(0, version.version.indexOf('+'))}`
```

Comment:
I'm wondering if I should check to see if the `version.version` string includes a '+' instead of whether the version `isEnterprise`. From my understand it can be `+hsm`, `+prem` (I've seen this somewhere but not sure!).

Reply:
FWIW `isEnterprise` does just check if the string includes a `+` 😅

Reply:
I noticed in when I updated the service File in this PR https://github.com/hashicorp/vault/pull/20479/files#diff-7887446e62a10f97f9cdcee5b4341aabc36fc0b3b56ff5423ef68b782ae01835L36

Reply:
ahh, i forgot to check the version service `isEnterprise` getter. Thanks for pointing that out!